### PR TITLE
Docs6877/add azure arm instructions

### PR DESCRIPTION
**Pull request by buraizu**
### What does this PR do? What is the motivation?
- Add instructions for setting up the Datadog Azure integration with ARM templates
- Add screenshots of the new process

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/integrations/guide/azure-manual-setup.md [hunk 1]
```diff
@@ -135,6 +135,49 @@ azure role assignment create --objectId <OBJECT_ID> -o "Monitoring Reader" -c /s
 
 ### Integrating through the Azure portal
 
+{{< tabs >}}
+{{% tab "ARM template" %}}
+
+1. In the Azure integration tile, select **Configuration** > **New App Registration** > **Using Azure Portal**.
+
+2. Select Management Group (Auto-Discover) or Individual Subscriptions. If you select Management Group, Datadog automatically discovers and monitors any subscriptions within that selected scope, including any subscriptions created in the future. You must have the owner role in the Management Group selected.
+
```
Comment (ashwingandhi-ddog) [inline content/en/integrations/guide/azure-manual-setup.md:144]: Also, If they select `Individual Subscriptions` then they need have the `owner` role in the subscriptions they selected.

File: content/en/integrations/guide/azure-manual-setup.md [hunk 2]
```diff
@@ -135,6 +135,49 @@ azure role assignment create --objectId <OBJECT_ID> -o "Monitoring Reader" -c /s
...
+9. Copy the value of the client secret, and click the close **(X)** button in the top-right corner of the screen.
+
+10. Paste the value of the client secret in the corresponding field on the template, and click **Next**.
+
+11. Provide a Datadog API key and Datadog Application key value in the corresponding fields. If you launched the template from the Azure integration page in Datadog, you can copy the keys provided there. Otherwise, you can find your API and App keys in the Access section of the Organization Settings.
+
+   **Note**: If you've selected to monitor individual subscriptions rather than a management group, select the subscriptions to monitor from the **Subscriptions to monitor** dropdown.
+
+12. Select your Datadog site, as well as any other integration configuration options, such as host filters and whether to collect resources for [Cloud Security Management][17].
```
Comment (brett0000FF) [inline content/en/integrations/guide/azure-manual-setup.md:171]:
```suggestion
12. Select your Datadog site and other integration configuration options, such as host filters and whether to collect resources for [Cloud Security Management][17].
```

File: content/en/integrations/guide/azure-manual-setup.md [hunk 3]
```diff
@@ -135,6 +135,49 @@ azure role assignment create --objectId <OBJECT_ID> -o "Monitoring Reader" -c /s
...
+
+13. Click **Next** or **Review + create**, then click **Create**.
```
Comment (brett0000FF) [inline content/en/integrations/guide/azure-manual-setup.md:173]: What is the difference between Next or Review + create? We should probably just list the happy path we want users to take, but I may be missing context.

**Review by brett0000FF — APPROVED** [review]
Looks good! I just left a few suggestions for you to consider.

File: content/en/integrations/guide/azure-manual-setup.md (hunk 3 above)
Comment (buraizu) [reply content/en/integrations/guide/azure-manual-setup.md]: Thanks for flagging, there is some redundancy in the Azure UI here ([screenshot](https://a.cl.ly/L1uyo9ov)). You make a good point about keeping it simple, updated in [c0b67e1](https://github.com/DataDog/documentation/pull/21084/commits/c0b67e12745881e3395854c6635784953e219c84)

**Pull request merged**